<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1939</width>
    <height>1166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(85, 125, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>861</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QTableWidget" name="tableWidgetDatosExcel">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <column>
        <property name="text">
         <string>RAT</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>OPERATOR</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CHANNEL</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IMEI</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IMSI</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TMSI</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MS POWER</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TA</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>LAST LAC</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>NAME</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>HITS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>DATA-TIME</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QTextEdit" name="textEditBuscarDatos">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonBuscarDato">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(63, 117, 194, 200), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset resource="../Recursos/qrc.qrc">
      <normaloff>:/newPrefix/icons8-google-web-search-400.png</normaloff>:/newPrefix/icons8-google-web-search-400.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>311</width>
      <height>165</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonVerDatosIncidentales">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Ver Datos Incidentales</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Recursos/icons8-big-data-150.png</normaloff>../Recursos/icons8-big-data-150.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonVerTablaOriginal">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Ver Tabla original</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Recursos/icons8-big-data-150.png</normaloff>../Recursos/icons8-big-data-150.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFiltrarDatos">
       <property name="font">
        <font>
         <family>Corbel</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Filtrar Datos</string>
       </property>
       <property name="icon">
        <iconset resource="../Recursos/qrc.qrc">
         <normaloff>:/newPrefix/icons8-ver-80.png</normaloff>:/newPrefix/icons8-ver-80.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonRATS">
         <property name="font">
          <font>
           <family>Corbel</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>RAT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonOperadores">
         <property name="font">
          <font>
           <family>Corbel</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Operador</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>Corbel</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>IMSI</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <family>Corbel</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>IMEI</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelRATContador">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelIMEIDatos">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelIMSIDatos">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelOperadorDatos">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonAsignarNombresIMEI">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Asignar Nombres a IMEI</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Recursos/icons8-ayuda-100.png</normaloff>../Recursos/icons8-ayuda-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGuardarDatos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>289</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Guardar Datos</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Recursos/upload.png</normaloff>../Recursos/upload.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Imagenes/Logo.png);
image: url(:/newPrefix/Logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Imagenes/Logo.png);
image: url(:/newPrefix/Logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAnalisisHorario">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>70</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0149254 rgba(85, 125, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Análisis Horario</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Recursos/icons8-tiempo-400.png</normaloff>../Recursos/icons8-tiempo-400.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1939</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatos">
    <property name="title">
     <string>Datos</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIr_A_Principal"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuDatos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIr_A_Principal">
   <property name="text">
    <string>Ir A Principal </string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar </string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Recursos/qrc.qrc"/>
 </resources>
 <connections/>
</ui>
